notifications/provision.go

```diff
@@ -18,7 +19,7 @@ func Provision(app, webHookURL string) error {
 	c := config.ScalingoClient()
 	params, err := c.NotificationProvision(app, webHookURL)
 	if err != nil {
-		return errors.New("Notification " + webHookURL + " can't be enabled because you have already an other notification of the same type enabled!")
+		return errors.New("Notification to " + webHookURL + " can't be enabled because you have already an other notification of the same type " + getType(webHookURL) + " enabled!")
```

Comment:
You should use standard validation error handling which reads the error from the server